did/.htaccess

```diff
@@ -2,5 +2,13 @@ Header set Access-Control-Allow-Origin *
 Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
 Options +FollowSymLinks
 RewriteEngine on
-RewriteRule ^$ https://w3c-ccg.github.io/did-spec/ [R=302,L]
+RewriteRule ^$ https://www.w3.org/TR/did-core/ [R=302,L]
+
+# TODO: fix this once did core is published
+
 RewriteRule ^v(.+)$ https://w3c-ccg.github.io/did-spec/contexts/did-v$1.jsonld [R=302,L]
```

Comment:
I know this isn't in this PR, but it seems wrong, no? We want to point to did-spec-registries now?

Reply:
Yes, that's what I meant, thanks.